install/master-setup.sh

```diff
@@ -91,15 +91,20 @@ if [[ "${MASTER_ID}" != "" ]]; then
 fi
 
 # stop the Cluster-Insight minion collectors.
-echo "stop the Cluster-Insight replication controller and minions"
-exec_command kubectl stop -f ${REP_CONTROLLER}
+rc_output="$(kubectl get rc | fgrep ${IMAGE})"
+if [[ -n $rc_output ]] ; then
```

Comment:
You have to be careful when rc_output is empty. You may get a syntax error if it is empty.
Please change the condition to:
if [[ -n "${rc_output}" ]] ; then